summary.qmd

```diff
@@ -0,0 +1,3 @@
+# Summary
+
+In summary, this book has no content whatsoever.
```

Comment:
🤔

Reply:
I.. have no recollection of writing this? 😆